src/client.cpp

```diff
@@ -523,6 +523,62 @@ GAME_ERROR CGameLibRetro::GetRichPresenceEvaluation(char* evaluation, size_t siz
   return GAME_ERROR_NO_ERROR;
 }
 
+GAME_ERROR CGameLibRetro::ActivateAchievement(unsigned cheevo_id, const char* memaddr)
+{
+  CCheevos::Get().ActivateAchievement(cheevo_id, memaddr);
+  return GAME_ERROR_NO_ERROR;
```

Comment:
you should change `ActivateAchievement` to return a boolean value and return `GAME_ERROR_FAILED` if it returns `false` like the other methods